doc/guide/transactions-and-threading.rst

```diff
@@ -0,0 +1,394 @@
+============================
+Transactions and concurrency
+============================
+
+.. contents::
+
+`Transactions <https://en.wikipedia.org/wiki/Database_transaction>`_
+are a core feature of ZODB.  Much has been written about transactions,
```

Comment:
[zope.sqlalchemy](https://github.com/zopefoundation/zope.sqlalchemy) is one I'm immediately aware of. There's also `repoze.sendmail`.